src/modules/table.marko

```diff
@@ -0,0 +1,945 @@
+<div id="table">
+    <section-header metadata=input.metadata name=input.name alpha="true"/>
+
+    <p>A table represents tabular data in rows and columns with incrementally enhanced feature set added on top for sorting, editing, etc.</p>
+
+    <p>Interactive elements should be tabbed into in correct sequence top down row by row and column by column based on the respective LTR or RTL mode.</p>
+
+    <h3>Basic Default Table</h3>
+    <div class="demo">
+        <div class="demo__inner">
+            <div class="table" tabindex="0">
```

Comment:
Related article on adding tabindex=0 to the container: 

https://adrianroselli.com/2017/11/a-responsive-accessible-table.html#ResponsiveScrollingKeyboard

We'll probably need to follow this advice here.